com.ozmartians.VidCutter.json

```diff
@@ -0,0 +1,202 @@
+{
+    "app-id": "com.ozmartians.VidCutter",
```

Comment:
http://ozmartians.com/ doesn't seem to be an active domain? It is nice to be something you clearly control to ensure no future collisions.

Reply:
@TingPing 

Its my domain but doesn't have a base webpage is all. A few subdomains and personal email domain etc.... Its in my control and will continue to be renewed so no need for concerns there unless I am misunderstanding you.

The VidCutter app, for example,  is subdomained from there at https://vidcutter.ozmartians.com but its just a redirect to GitHub. I could make the base domain simply redirect to my GitHub profile page I guess.

Reply:
Ok thats fine.